web/forms.py

```
@@ -1787,3 +1792,42 @@ class StudyGroupForm(forms.ModelForm):
+            "stripe_monthly_price_id": TailwindInput(),
+            "stripe_yearly_price_id": TailwindInput(),
+            "is_active": TailwindCheckboxInput(),
+            "is_popular": TailwindCheckboxInput(),
+            "order": TailwindNumberInput(attrs={"min": "0"}),
+        }
+        help_texts = {
+            "features": "Enter the features for this membership plan, one per line.",
+            "is_popular": "Mark this plan as popular to highlight it on the pricing page.",
+            "stripe_monthly_price_id": "Stripe Price ID for monthly billing",
+            "stripe_yearly_price_id": "Stripe Price ID for yearly billing",
+        }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)